exercises/twelve-days/twelve_days_test.py

```diff
@@ -128,23 +130,137 @@ def test_verse12(self):
                     "four Calling Birds, "
                     "three French Hens, "
                     "two Turtle Doves, "
-                    "and a Partridge in a Pear Tree.\n")
-        self.assertEqual(verse(12), expected)
+                    "and a Partridge in a Pear Tree."]
+        self.assertEqual(recite(12, 12), expected)
+
 
-    def test_multiple_verses(self):
-        expected = ("On the first day of Christmas my true love gave to me, "
-                    "a Partridge in a Pear Tree.\n\n"
+class LyricsTests(unittest.TestCase):
+    def test_first_three_verses_of_the_song(self):
+        expected = ["On the first day of Christmas my true love gave to me, "
```

Comment:
Since there are other tests verifying correct output from `recite(n, n)`, I think the following would be adequate:

```Python
expected = [recite(n, n) for n in range(1, 4)]
```

This also removes the need for large redundant text blocks.

Reply:
@cmccandless Pretty nice trick! But since `recite` returns a list of one string, we should capture its first element:

```python
expected = [recite(n, n)[0] for n in range(1, 4)]
```

Reply:
On the idea of nice tricks...

```Python
expected = sum([recite(n, n) for n in range(1, 4)], [])
```

Not really any neater than what you're already doing, and it's not as readable, so please don't use that for this exercise.